packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -69,6 +69,64 @@ class SemanticsTag {
   String toString() => '$runtimeType($name)';
 }
 
+/// A custom accessibility action.
+/// 
+/// Custom accessibility actions can be provided to make complex user
+/// interactions more accessible. A drag and drop list, for example, may
+/// require a user to press and hold an item to move it.  This may be
+/// difficult for a user interacting with your app using a hardware switch. A
+/// custom accessibility action allows you to specifiy actions such as 
+/// "move up" and "move down" on items in this list.
+/// 
+/// In Android, these actions are presented in the local context menu. In iOS,
+/// these are in the radial context menu.
+class CustomAccessibilityAction {
+
+  /// Creates a new [CustomAccessibilityAction].
+  /// 
+  /// [label] must not be null or the empty string.
```

Comment:
consider starting with "The [label] argument must..." to avoid starting with a lowercase letter.

Reply:
Fixed